### DISCUSSION TITLE
호연 파티 저녁 메뉴 투표

### DISCUSSION THREAD

**Discussion by iamyanghee** [Melon-Shake/API#30] [democracy]
@mews2000 @json9101 @bksooon @Lucete28 @jungssg @iamyanghee 

투표해주고, 괜찮은 장소 있으면 달아주셈

**Comment by iamyanghee** [comment]
### 이가네양꼬치 

2팀도 회식 여기서 했다함. 
성남서 유명해서 체인 생겼다 함.
맛있다 함.
양왕꼬치 1.8만원

[카카오맵] 이가네양꼬치
서울 서초구 반포대로22길 89 1층 (서초동) 
http://kko.to/YUcMHDQ3cp

**Reply by bksooon** [reply]
무한리필임?

**Reply by Lucete28** [reply]
1인분입니다

**Comment by iamyanghee** [comment]
### 히츠지야 교대점

4.5만원인가에 양고기 여러부위 무제한
2만원에 90분동안 맥주 무제한
분위기 좋음
6명 갈꺼면 예약하기

[카카오맵] 히츠지야 교대점
서울 서초구 반포대로28길 73 1층 (서초동) 
http://kko.to/hBfaCNMeqD

**Reply by bksooon** [reply]
비싸여

**Comment by Lucete28** [comment]
무한 숯불 양꼬치 
1인 31000원 
양꼬치, 훠궈 무한리필
술 시키면 서비스줌
https://naver.me/5lCOmUBI
